Libraries/Components/Pressable/Pressable.js

```diff
@@ -52,6 +52,7 @@ type Props = $ReadOnly<{|
   accessibilityValue?: ?AccessibilityValue,
   accessibilityViewIsModal?: ?boolean,
   accessible?: ?boolean,
+  'aria-live'?: ?('none' | 'polite' | 'assertive' | 'off'),
```

Comment:
`'none'` is not a valid value for `aria-live`

Reply:
Updated the requested changes.